source/docs/yearly-overview/yearly-changelog.rst

```diff
@@ -197,3 +197,7 @@ Supported Operating Systems and Architectures:
 ## AdvantageScope
 
 - Update to (2025 AdvantageScope)[https://docs.advantagescope.org/whats-new]
+
+## Choreo
+
+New for 2025 Choreo will be bundled in the installer! Choreo is an application for creating time optimal autonomous paths. (Read more here)[https://sleipnirgroup.github.io/Choreo/].
```

Comment:
Everything on this page is new for 2025, so that's redundant. It also should be present or past tense, not future tense, because it will be true by the time most people read this.